changelog/unreleased/4484

```diff
@@ -0,0 +1,4 @@
+Enhancement - Added text labels and active indicator for BottomNavigationView
```

Comment:
Actually, the active indicator was already added, so let's just mention the labels in this case 😄